modules/standard/timers/timer_controller.py

```diff
@@ -97,6 +97,8 @@ def timer_remove_cmd(self, request, _, timer_name):
              description="Add a timer")
     def rtimer_add_cmd(self, request, _, start_time, repeating_time, timer_name):
         timer_name = timer_name or self.get_timer_name(request.sender.name)
+        if repeating_time < 120:
```

Comment:
If we are going to have a limit here, I think we should set it to 60s.  There might be instances where they want a repeating timer every minute.  It is better to make the limit too low, than too high.

Reply:
changed it to 60 seconds, in the following commit.